lib/vfscore/fops.c

```diff
@@ -45,6 +45,13 @@ int vfs_close(struct vfscore_file *fp)
 	int error;
 
 	vn_lock(vp);
+	/* This is called when there are no more strong references to this file
```

Comment:
Maybe put also a `FIXME` comment. As we discussed, the sync operation should actually happen on the fs driver during unmount. This is a workaround for now.

Reply:
This should actually be fine and make sense. If there are no more references to an open file structure, it makes sense to sync, right? CC @andreittr